applications/luci-app-vpnbypass/root/usr/share/rpcd/acl.d/luci-app-vpnbypass.json

```diff
@@ -2,38 +2,16 @@
 	"luci-app-vpnbypass": {
 		"description": "Grant UCI and file access for luci-app-vpnbypass",
 		"read": {
-			"cgi-io": [
-				"exec"
-			],
-			"file": {
-				"/usr/lib/opkg/status": [
-					"read"
-				],
-				"/etc/init.d/vpnbypass *": [
-					"exec"
-				],
-				"/etc/init.d/dnsmasq restart *": [
-					"exec"
-				],
-				"/usr/bin/grep *": [
-					"exec"
-				],
-				"/usr/sbin/grep *": [
-					"exec"
-				],
-				"/usr/sbin/iptables *": [
-					"exec"
+			"ubus": {
+				"luci.vpnbypass": [
+					"getInitList",
+					"setInitAction",
```

Comment:
The `setInitAction` method has side-effects (allows altering system state), please move it from the `"read": { ... }` to the `"write": { ... }` scope.